R/pull.R

```diff
@@ -107,11 +107,10 @@ reduce_all_outcome_names <- function(resamples) {
   }
 
   if (n_unique > 1L) {
-    rlang::warn(paste0(
-      "More than one set of outcomes were used when tuning. ",
-      "This should never happen. ",
```

Comment:
This was weirdly aggro. Ahh, me in July 2021. I blame the 'rona.

Reply:
hahahaha i feel like this is the most likely error i see when i introduce bugs in the grid code paths